flink-runtime/src/main/java/org/apache/flink/runtime/state/heap/HeapSnapshotStrategy.java

```diff
@@ -60,8 +60,7 @@
  * Base class for the snapshots of the heap backend that outlines the algorithm and offers some hooks to realize
```

Comment:
Here the java doc said that "Base class", but there is no subclass for `HeapSnapshotStrategy`, do you think we should we should update the java doc also?